ft.py

```diff
@@ -0,0 +1,75 @@
+import pandas as pd
```

Comment:
do we want to have this file in the repository?

Reply:
removed